launch-RA2.bat

```diff
@@ -0,0 +1 @@
+OpenRA\OpenRA.exe SupportDir=D:/OpenRA-RA2-ipchote/
```

Comment:
Placeholder. See the linked upstream issue - atm I left it in as a NTM and as a testcase for upstream to look at.